wabi/src/main/kotlin/com/wap/wabi/band/controller/BandController.kt

```diff
@@ -0,0 +1,21 @@
+package com.wap.wabi.band.controller
+
+import com.wap.wabi.band.service.BandService
+import com.wap.wabi.common.payload.response.Response
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+
+@Controller
+@RequestMapping("/api/bands/")
+class BandController(
+    private val bandService: BandService
+) {
+    @GetMapping("{bandId}/students")
+    fun getBandStudents(@PathVariable bandId: Long) : ResponseEntity<Response>{
+        return ResponseEntity(bandService.getBandStudents(bandId = bandId), HttpStatus.OK)
+    }
```

Comment:
```suggestion
    fun getBandStudents(@PathVariable bandId: Long) : ResponseEntity<Response> {
        return ResponseEntity(bandService.getBandStudents(bandId = bandId), HttpStatus.OK)
    }
```
IntelliJ Settings의 Editor -> Code Style에서 Scheme 형식을 통일할 필요가 있어 보이네요!